docs/tools/phpstorm.md

```diff
@@ -27,4 +27,39 @@ That's it. You can now use this code style on all future CiviCRM-related project
 
 TODO
 
+## Running automated tests from within PhpStorm {:#testing}
+
```

Comment:
this makes sense, there were some changes from the sprint that I hadn't seen so didn't notice this dedicated page before.